MdeModulePkg/Universal/CapsuleRuntimeDxe/CapsuleReset.c

```diff
@@ -24,5 +24,9 @@ IsPersistAcrossResetCapsuleSupported (
   VOID
   )
 {
+  if (!DasharoCapsulesCanPersistAcrossReset()) {
+    return FALSE;
+  }

```

Comment:
Can `FeaturePcdSet()` be used instead of adding this check here?

Reply:
There is no such function, feature flags are fixed at build time.  `AutoGen.h` contains:
```c
//#define _PCD_SET_MODE_BOOL_PcdSupportUpdateCapsuleReset  ASSERT(FALSE)  // It is not allowed to set value for a FIXED_AT_BUILD PCD
```

Reply:
Right, it did look strange to me, but such function [is mentioned in the edk2 docs](https://github.com/tianocore-docs/edk2-ModuleWriteGuide/blob/master/3_module_development/32_creating_a_module.md#table-5-pcd-access-functions). I guess we have to live with it, maybe during upstreaming someone will propose another solution.

Reply:
Upstream shouldn't see this commit because it depends on many other Dasharo-specific modifications.